dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --filesystem=home/.var/app/io.mpv.Mpv/config/mpv/scripts:create
+  - --env=LC_NUMERIC=C
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --own-name=org.mpris.MediaPlayer2.mpv.*
+  - --talk-name=org.gnome.SessionManager
+
+cleanup:
+  - '*.la'
+  - '*.a'
+
+environment:
+  WEBKIT_DISABLE_DMABUF_RENDERER: 1
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/Fredolx/open-tv/releases/download/v1.0.0/flatpak.deb
+        sha256: 62555af870ee10e0e363bc457ca280046dc19094cbb4d96b7e5770931b28dcaf
+        only-arches: [x86_64]
```

Comment:
this is not enough to restrict the build to x86_64 only. Rest of the manifest is being built for all arches.

https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson